index.html

```diff
@@ -433,6 +443,20 @@ <h2><dfn>disconnect()</dfn> method</h2>
         <a data-cite="!WHATWG-DOM#context-object">context object</a>'s
         <a>observer buffer</a>.</p>
       </section>
+      <section>
+        <h2><dfn>supportedEntryTypes</dfn> attribute</h2>
+        <p>The user agent must maintain <dfn>supported entry types</dfn>, a list
+        of strings representing the entry types which the user agent supports
+        for the <a>PerformanceObserver</a> interface. This list is populated by
+        specs that define new entry types via the hook to
+        <a href="#register-performance-entry-type"></a>.</p>
+        <p>The <a>supportedEntryTypes</a>'s attribute getter must return a copy
+        of the user agent's <a>supported entry types</a>, sorted in alphabetical
+        order.</p>
+        <p class="note">This attribute allows web developers to easily know
+        which entry types are supported by the user agent and to discover the
+        entry types that exist and are supported by the user agent.</p>
```

Comment:
Seems like we're repeating ourselves three times here? Could we simplify to..

> This attribute allows web developers to easily know which entry types are supported by the user agent.

Reply:
Done